src/test/runTest.sh

```diff
@@ -2,6 +2,7 @@
 
 pushd `dirname $0`
 export LD_LIBRARY_PATH=/opt/circonus/lib/amd64:/opt/circonus/lib
+export DYLD_LIBRARY_PATH=/opt/circonus/lib/amd64:/opt/circonus/lib:..
```

Comment:
It worked before only when libcircllhist.so was installed when this script is run. I guess it doesn't interfere, but  MacOS and other Unixes should probably behave the same.